modules/context/xsrf.go

```diff
@@ -28,69 +28,67 @@ import (
 	"time"
 )
 
-// Timeout represents the duration that XSRF tokens are valid.
+// CsrfTokenTimeout represents the duration that XSRF tokens are valid.
 // It is exported so clients may set cookie timeouts that match generated tokens.
-const Timeout = 24 * time.Hour
+const CsrfTokenTimeout = 24 * time.Hour
+const CsrfTokenRegenerationDuration = 1 * time.Minute
```

Comment:
> This seems far too frequent. Perhaps this should be hourly?

It doesn't matter, it's just a simple hash when generating the token, doesn't cost more resources than other logic.

And the generated token is valid for the next 24 hours, so there should be no problem.

Indeed we can keep generating new tokens on every request, there will still be no performance problem. I just introduced this mechanism in case someone doesn't like generating the tokens again and again.